pkgs/applications/editors/emacs/elisp-packages/manual-packages/yes-no/default.nix

```diff
@@ -1,19 +1,36 @@
-{ lib, fetchurl, trivialBuild }:
+{ lib
+, fetchFromGitHub
+, melpaBuild
+, writeText
+}:
 
-trivialBuild {
+let
   pname = "yes-no";
-  version = "2017-10-01";
-
-  src = fetchurl {
-    url = "https://raw.githubusercontent.com/emacsmirror/emacswiki.org/143bcaeb679a8fa8a548e92a5a9d5c2baff50d9c/yes-no.el";
-    sha256 = "03w4wfx885y89ckyd5d95n2571nmmzrll6kr0yan3ip2aw28xq3i";
+  ename = "yes-no";
+  version = "20240107.122";
+  src = fetchFromGitHub {
+    owner = "emacsmirror";
+    repo = "emacswiki.org";
+    rev = "02dbe9aa48c735cfcc7e977ea8fe512328666cad";
```

Comment:
Using the old commit seems more reasonable to me because `yes-no.el` has not changed since then.

Reply:
I am using the most recent commit here, since it is easier (no need for pushing that specific commit) and goes in line with the idea of "track the most recent commit".

Further, it can be more useful if anyone is interested on other single-file packages from ewiki.